src/sentry/web/urls.py

```diff
@@ -146,6 +148,16 @@
         ErrorPageEmbedView.as_view(),
         name="sentry-error-page-embed",
     ),
+    re_path(
+        r"^organizations/(?P<organization_slug>[^/]+)/toolbar/iframe$",
```

Comment:
needs a trailing slash in the url

```suggestion
        r"^organizations/(?P<organization_slug>[^/]+)/toolbar/iframe/$",
```